extensions/functions_datetime.yaml

```diff
@@ -35,7 +35,12 @@ scalar_functions:
         return: i64
   -
     name: "add"
-    description: Add an interval to a date/time type.
+    description: >-
+      Add an interval to a date/time type.
+
+      Timezone strings must be as defined by IANA timezone database (https://www.iana.org/time-zones).
+      Examples: "Pacific/Marquesas", "Etc/GMT+1".
+      If timezone is not present then UTC is assumed. If timezone is invalid an error is thrown.
```

Comment:
How would it be possible now for the timezone to not be present?

Reply:
By "present" I mean provided as a parameter. I'll rephrase.

Reply:
In Substrait, currently, arguments are always required and only options are optional.  So there is no way to call this function without including the timezone as a parameter.  I think this is fine, and we should just remove the "default to UTC" wording.  If a producer wants to default to UTC then they can do so very easily.  However, the plan should be as simple and explicit as reasonably possible.

Reply:
That makes sense. Changed `If timezone is not provided as parameter then UTC is assumed. If timezone is invalid an error is thrown.` to `If timezone is invalid an error is thrown.`